frontend/app/src/people/widgetViews/summaries/wantedSummaries/lib.ts

```diff
@@ -0,0 +1,35 @@
+import { CodingLanguageLabel } from 'people/interfaces';
+
+type Props = {
+  title: string;
+  labels?: Array<CodingLanguageLabel>;
+  ownerPubkey: string;
+  issueCreated: string;
+};
+export const getTwitterLink = ({ title, issueCreated, ownerPubkey, labels }: Props) => {
+  const bountyParams = {
+    owner_id: ownerPubkey,
+    created: issueCreated
+  };
+  const bountyUrl = new URL('https://community.sphinx.chat/tickets');
```

Comment:
this may change depeneding on where it is deployed we probably don't want to hard code this

Reply:
I'm going to merge it for now though

created this issue to resolve this
https://github.com/stakwork/sphinx-tribes/issues/687